util/containers/map_test.go

```diff
@@ -0,0 +1,49 @@
+package containers
+
+import (
+	"testing"
+)
+
+func TestMap(t *testing.T) {
+	var m Map[int, int]
+	m.Store(13, 31)
+	m.Store(11, 12)
+	m.Store(11, 11)
+	m.Store(42, 24)
```

Comment:
would be fun to have at least all stores in separate threads. We can do Store(42,24) and Delete(42) in the same thread.